src/EventSubscriber/CommentNotificationSubscriber.php

```diff
@@ -50,10 +50,11 @@ public function onCommentCreated(CommentCreatedEvent $event): void
             '_fragment' => 'comment_'.$comment->getId(),
         ], UrlGeneratorInterface::ABSOLUTE_URL);
 
+        // See https://symfony.com/doc/current/translation.html
         $subject = $this->translator->trans('notification.comment_created');
         $body = $this->translator->trans('notification.comment_created.description', [
-            '%title%' => $post->getTitle(),
-            '%link%' => $linkToPost,
+            'title' => $post->getTitle(),
+            'link' => $linkToPost,
```

Comment:
IIRC it's still best practice to use `%%` for translation params, at least the link you provided uses `%link%` instead of `link`, so why should we remove it?

Reply:
I am reading in the link:

// prints "Hello Fabien!"
echo $translator->trans('say_hello', ['name' => 'Fabien']);

// prints "Hello Symfony!"
echo $translator->trans('say_hello', ['name' => 'Symfony']);

Reply:
Hm, are you sure you're on the "current"? i.e. https://symfony.com/doc/current/translation.html

I don't see "say_hello" at all, but see:
<img width="842" alt="Screenshot 2021-12-20 at 15 37 36" src="https://user-images.githubusercontent.com/3317635/146775578-b92957eb-9fa0-43d5-bf69-422dda889f43.png">

Reply:
@bocharsky-bw now I got it ...
I am on the https://symfony.com/doc/current/translation/message_format.html (as in PR description) not the one you answered me (as in PR code for documentation of translations)

And the subpart at https://symfony.com/doc/current/translation/message_format.html#message-placeholders

for ICU translations messages => **as it is what this demo app uses**
<img width="834" alt="Capture d’écran 2021-12-20 à 14 45 53" src="https://user-images.githubusercontent.com/1358361/146776859-001296eb-b0f2-4031-8809-c4b2d7a6e150.png">

perhaps I should put both links?
thank you

Reply:
Aha, now I see. Changes look good then, but I'm not sure about the links. We use translations all over the project code, and having the link only in this spot sounds incomplete to me, but on the other hand having the link in all the spots where we use translations is overkill probably. Maybe we don't need that link at all?

Also, if we want to get rid of `%%` in this spot, probably we should do the same on other spots too?